### DISCUSSION TITLE
Explain identity brokering as concept

### DISCUSSION THREAD

**Discussion by mffap** [zitadel/zitadel#1983] [Ideas]
The concept of identity brokering is only little explained but a bit too complex to understand without further information. Generally the concept is also not easy to find as most vendors call it differently (external identity providers, account linking, connections, identity broker...). 

We should document the big picture, related concepts, relationships of the objects etc., independent of how identity brokering is configured within ZITADEL.

Some resources, I found useful:
- https://ldapwiki.com/wiki/Identity%20Broker
- https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.0/html/server_administration_guide/identity_broker
- https://developer.okta.com/docs/concepts/identity-providers/

**Comment by gc-ss** [comment]
Target: https://docs.zitadel.ch/docs/guides/usage/identity-brokering/
